src/methods.jl

```diff
@@ -179,3 +179,63 @@ function trimview(frame::AbstractArray, idxs)
 
     return selectdim(frame, d, complement_idxs)
 end
+
+
+"""
+    crop(frame::AbstractArray, shape; force_odd = true)
+
+Crops the image to the size specified asumming center as the anchor.
```

Comment:
take a look at the project notes for more ideas for documentation.

Reply:
I have added a few docstrings!